framework/helpers/Url.php

```diff
@@ -15,4 +15,35 @@
  */
 class Url extends BaseUrl
 {
+    /**
+     * Custom function to emulate http_build_query. The goal is to avoid getting numeric values bettween '[]'.
+     * For example:
+     * - http_build_query(['language' => ['en', 'ca']])           -> 'language%5B0%5D=en&language%5B1%5D=en'
+     * - custom_http_build_query(['language' => ['en', 'ca']])    -> 'language%5B%5D=en&language%5B%5D=ca'
+     *
+     * @param array  $params Array of values to convert into http query string
+     * @param string $inheritKey String used when array has more than one dimesion.
+     *                           We pass the our actual key to our child array.
+     * @return array
+     */
+    public static function custom_http_build_query(Array $params, $inheritKey = '') {
+        $response = [];
+        foreach($params as $key => $value) {
+            if(!empty($inheritKey)) {
+                if(is_numeric($key)) {
+                    $key = '';
+                }
+                $key = "{$inheritKey}[{$key}]";
+            }
+            if(is_array($value)) {
```

Comment:
I wouldn't add `get_object_vars`, let's just assume the object is iterable, that is it would work in `foreach`. It's just an idea, we need to keep it simple.